theseus/src/api/profile/mod.rs

```diff
@@ -1043,18 +1043,26 @@ pub async fn create_mrpack_json(
     })
 }
 
-fn sanitize_loader_version_string(s: &str) -> &str {
-    // Split on '-'
-    // If two or more, take the second
-    // If one, take the first
-    // If none, take the whole thing
-    let mut split: std::str::Split<'_, char> = s.split('-');
-    match split.next() {
-        Some(first) => match split.next() {
-            Some(second) => second,
-            None => first,
-        },
-        None => s,
+fn sanitize_loader_version_string(s: &str, loader: PackDependency) -> &str {
+    match loader {
+        // Split on '-'
+        // If two or more, take the second
+        // If one, take the first
+        // If none, take the whole thing
+        PackDependency::Forge
+        | PackDependency::FabricLoader
+        | PackDependency::Minecraft => {
```

Comment:
I don't actually see why this should be done for any pack dep other than Forge. For Minecraft, we don't want 1.19.4-rc1 being turned into 1.19.4. For Fabric, I don't think they even use dashes at all. So really this should just be a special case for Forge.

Reply:
agreed

Reply:
Done, makes sense